.github/workflows/plugin-check.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         working-directory: ${{ github.event.repository.name }}
 
       - name: Run plugin check
-        uses: wordpress/plugin-check-action@v1
+        uses: wordpress/plugin-check-action@70f38272b2beee386e973f319591baa2fc7dff16 # v1.1.2
```

Comment:
Noting that I think this one is still updating fairly regularly, but probably something we're ok trailing a bit on and just updating occasionally as they hit major updates